WindowsClient/FileViewer.xaml.cs

```diff
@@ -15,11 +15,12 @@
 
 namespace WindowsClient
 {
-    /// <summary>
-    /// Interaction logic for FileViewer.xaml
-    /// </summary>
+    // [Class.FileViewer] - A Gui to show file contents.
     public partial class FileViewer : Window
     {
+        /// <summary> To create a Viewer in future.
+        /// 
```

Comment:
Missing fields (i suppose you just wrote "to create a viewer in future" because you'll finish this later)